elliottlib/cli/attach_cve_flaws_cli.py

```diff
@@ -19,6 +20,9 @@
 @click.option('--advisory', '-a', 'advisory_id',
               type=int,
               help='Find tracker bugs in given advisory')
+@click.option('--id', default=None, multiple=True,
```

Comment:
Would like to call the parameter `ids`:
```suggestion
@click.option('--id', 'ids', multiple=True,
```

Note that `default` has no effects on a multi option. If `--id` is not given, the value will be `()` (empty tuple).